src/python/CRABClient/Commands/status.py

```diff
@@ -326,6 +327,9 @@ def printTaskInfo(self, crabDBInfo, username):
         if schedd:
             msg = "Grid scheduler:\t\t\t%s" % schedd
             self.logger.info(msg)
+        if twname:
+            msg = "Task Worker:\t\t\t%s" % twname
+            self.logger.info(msg)
```

Comment:
I prefer to not print this information in the crab status output and keep it only in the log, i.e. replace `self.logger.info` with `self.logger.debug`, in order to prevent user confusion with meaningless lines.
Same for the schedd name [three lines above](https://github.com/dmwm/CRABClient/pull/4775/files#diff-ff67af59351a4436cecb26ca2051f7b8R329).